lib/et/src/et_collector.erl

```diff
@@ -720,7 +810,17 @@ iterate(Handle, Prev, Limit) ->
 %% Fun = fun(Event, Acc) -> NewAcc
 %% Acc = NewAcc = term()
 %%----------------------------------------------------------------------
-
+-spec iterate(Handle, Prev, Limit, Fun, Acc) -> NewAcc when
+      Handle :: CollectorPid | table_handle(),
+      CollectorPid :: pid(),
+      Prev :: first | last | term(),
+      Limit :: Done | Forward | Backward,
+      Done :: 0,
+      Forward :: pos_integer() | 'infinity',
+      Backward :: neg_integer() | '-infinity',
+      Fun :: fun((event(), Acc) -> NewAcc) | undefined,
+      Acc :: term(),
+      NewAcc :: term().
```

Comment:
I think it would be good to adopt the description to  say equivalent to instead of short for to adhere to the rest of the documentation and make a link to iterate/5.  I see that you only addressed the spec part in this PR so I do not know if your intention is to fix these kind of improvements in a second round after conversion?!

Reply:
This is a spec only update, the docs will have to be fixed in a future PR.